yt_dlp/extractor/goplay.py

```diff
@@ -0,0 +1,76 @@
+import json
+
+from .common import InfoExtractor
+from ..utils import int_or_none
+from .goplay_auth_aws import AwsIdp
+
+
+class GoPlayIE(InfoExtractor):
+    _VALID_URL = r'https?://(www\.)?goplay\.be/video/(?P<series>[^/]+)/(?P<season>[^/]+)(/(?P<episode>[^/]+)|)'
+
+    _NETRC_MACHINE = 'goplay'
+
+    _TESTS = [{
+        'url': 'https://www.goplay.be/video/de-container-cup/de-container-cup-s3/de-container-cup-s3-aflevering-2#autoplay',
+        'info_dict': {
+            'id': '9c4214b8-e55d-4e4b-a446-f015f6c6f811',
+            'ext': 'mp4',
+            'title': 'S3 - Aflevering 2',
+            'series': 'De Container Cup',
+            'season': 'Season 3',
+            'season_number': 3,
+            'episode': 'Episode 2',
+            'episode_number': 2,
+        },
+        'skip': 'This video is only available for registered users'
+    }]
+
+    def _initialize_pre_login(self):
+        self._id_token = None
+
+    def _perform_login(self, username, password):
+        self.report_login()
+        aws = AwsIdp(pool_id='eu-west-1_dViSsKM5Y', client_id='6s1h851s8uplco5h6mqh1jac8m')
+        self._id_token, _ = aws.authenticate(username=username, password=password)
+
+    def _real_initialize(self):
+        if not self._id_token:
+            raise self.raise_login_required(method='password')
+
+    def _real_extract(self, url):
+        if "#" in url:
```

Comment:
use single quotes everywhere

Reply:
Even better, I've removed this since `_download_webpage_handle` already does this and the changed code using `traverse_obj` no longer has issues if the `#` is still present in the `url` used in checks.